wapitiCore/language_sources/fr.po

```diff
@@ -732,6 +736,18 @@ msgstr "L'administrateur web doit supprimer manuellement les sauvegardes présen
 msgid "Blind SQL Injection description"
 msgstr "Tout comme les failles d'injection SQL classiques, les injections SQL en aveugle permettent à un attaquant d'altérer les requêtes normalement exécutées sur la base de données. Ce type de vulnérabilité est plus difficile à détecter en raison de l'absence de messages d'erreur renvoyés par l'application web."
 
+#: ../language/vulnerability.py:114
+msgid "CSP Configuration"
+msgstr "Configuration du CSP"
+
+#: ../language/vulnerability.py:115
+msgid "CSP \"{0}\" is missing"
+msgstr "CSP \"{0}\" est manquant"
+
+#: ../language/vulnerability.py:116
+msgid "CSP \"{0}\" value is not safe"
+msgstr "La valeur du CSP \"{0}\" n'est pas sécurisée"
+
```

Comment:
Translations for Content Security Policy description and solution are missing in fr.po and en.po

Reply:
Fixed